content-src/components/Sections/Sections.jsx

```diff
@@ -306,14 +306,32 @@ export class _Sections extends React.PureComponent {
     return sections;
   }
 
+  renderComponent(component) {
+    const {props} = component;
+    if (component.type === "TopicsHeader") {
+      const {topics} = this.props.Sections.find(section => section.id === "topstories") || {};
+      return (
+        <TopicsHeader
+          topics={topics}
```

Comment:
I'm using the old topics returned from the old topics endpoint, I imagine we want to lock this down to be new test data specific to our tests.